index.bs

```diff
@@ -6547,7 +6547,7 @@ At this time, one [=credential property=] is defined: the [=resident key credent
 
 This [=client extension|client=] [=registration extension=] and [=authentication extension=] allows a [=[RP]=] to evaluate outputs from a pseudo-random function (PRF) associated with a [=credential=]. The PRFs provided by this extension map from {{USVString}}s of any length to 32-byte {{ArrayBuffer}}s.
 
-When requested, if a [=credential=] is created on an [=authenticator=] that supports this extension then either one or two freshly-seeded PRFs are associated with it. If two PRFs are associated then, when outputs are requested, one PRF is evaluated if [=user verification=] is performed and the other is evaluated if only a [=test of user presence|user presence test=] is performed. It is the responsibility of the [=[RP]=] to set the {{PublicKeyCredentialRequestOptions/userVerification}} parameter accordingly: [=[RPS]=] SHOULD *consistently* use either the {{UserVerificationRequirement/required}} or {{UserVerificationRequirement/discouraged}} values of {{UserVerificationRequirement}} when using this extension, otherwise the outputs may vary for a given input. To avoid mistakes, the default value of {{UserVerificationRequirement/preferred}} is prohibited when using this extension during [=assertion=] because that could cause the PRF used to vary between operations if [=user verification=] is later enabled on an [=authenticator=].
+When requested, if a [=credential=] is created on an [=authenticator=] that supports this extension then either one or two freshly-seeded PRFs are associated with it. If two PRFs are associated then, when outputs are requested, one PRF is evaluated if [=user verification=] is performed and the other is evaluated if only a [=test of user presence|user presence test=] is performed. It is the responsibility of the [=[RP]=] to set the {{PublicKeyCredentialRequestOptions/userVerification}} parameter accordingly: [=[RPS]=] SHOULD *consistently* use either the {{UserVerificationRequirement/required}} or {{UserVerificationRequirement/discouraged}} values of {{UserVerificationRequirement}} when using this extension, otherwise the outputs may vary for a given input. For example, if this extension is used with {{PublicKeyCredentialRequestOptions/userVerification}} set to {{UserVerificationRequirement/preferred}} and an authenticator that doesn't have [=user verification=] configured, then later configuring user verification on that authenticator may cause the PRF outputs to change. Because of this, when creating a credential with this extension where {{PublicKeyCredentialRequestOptions/userVerification}} set to {{UserVerificationRequirement/preferred}}, the [=client platform=] SHOULD configure the authenticator for [=user verification=] if it is capable of it as that avoids the PRF changing in the future. At assertion time, [=[RPS]=] SHOULD check the UV bit in the [=flags=] to confirm that the expected PRF was evaluated.
```

Comment:
Two suggestions:

- Fix a typo: "where `userverification` _is_ set to `preferred`"
- Rewrite "UV bit in the flags" as "UV flag" and autolink both "UV" and "flag"

```suggestion
When requested, if a [=credential=] is created on an [=authenticator=] that supports this extension then either one or two freshly-seeded PRFs are associated with it. If two PRFs are associated then, when outputs are requested, one PRF is evaluated if [=user verification=] is performed and the other is evaluated if only a [=test of user presence|user presence test=] is performed. It is the responsibility of the [=[RP]=] to set the {{PublicKeyCredentialRequestOptions/userVerification}} parameter accordingly: [=[RPS]=] SHOULD *consistently* use either the {{UserVerificationRequirement/required}} or {{UserVerificationRequirement/discouraged}} values of {{UserVerificationRequirement}} when using this extension, otherwise the outputs may vary for a given input. For example, if this extension is used with {{PublicKeyCredentialRequestOptions/userVerification}} is set to {{UserVerificationRequirement/preferred}} and an authenticator that doesn't have [=user verification=] configured, then later configuring user verification on that authenticator may cause the PRF outputs to change. Because of this, when creating a credential with this extension where {{PublicKeyCredentialRequestOptions/userVerification}} set to {{UserVerificationRequirement/preferred}}, the [=client platform=] SHOULD configure the authenticator for [=user verification=] if it is capable of it as that avoids the PRF changing in the future. At assertion time, [=[RPS]=] SHOULD check the [=authData/flags/UV=] [=flag=] to confirm that the expected PRF was evaluated.
```

Reply:
(I editing this suggestion because I think the typo fix was done in the wrong place. Please search for "is set to" in the result and check that it makes sense.)

Reply:
Sorry, yes - good catch!